### PR TITLE
LPS-183961[Marketplace] Add Product Specification Label

### PR DESCRIPTION
**Pull request by juliocezar96**
[LPS-183961](https://issues.liferay.com/browse/LPS-183961)

### CODE REVIEW

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:bundle

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/commerce-catalogs/catalog.products.specifications.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
+[
+	{
+		"cpDefinitionExternalReferenceCode": "MP-TYPE",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/commerce-catalogs/catalog.products.specifications.json:3]:
```suggestion
		"cpDefinitionExternalReferenceCode": "MKT-TYPE",
```

Same for the others.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job, @juliocezar96. I added some suggestions. I also would like to ask you to take a look at the`ddm-templates` folder in your commit. Do you really need it?

**Comment by juliocezar96** [comment]
> Nice job, @juliocezar96. I added some suggestions. I also would like to ask you to take a look at the`ddm-templates` folder in your commit. Do you really need it?

@joanagrego 
I just sent the changes.

Yes, Brian asked to correct the codigo indentation in this pr https://github.com/brianchandotcom/liferay-portal/pull/134572#issuecomment-1542187100

**Comment by juliocezar96** [comment]
ci:test:sf

**Comment by juliocezar96** [comment]
ci:test:bundle

**Comment by joanagrego** [comment]
Resent --> [PR](https://github.com/pt-liferay-solutions/liferay-portal/pull/3960)

**Pull request closed without merging**